### ISSUE TITLE
You have not been authorized (2)

### ISSUE THREAD

**Issue by 0dayCTF** [kleo/evilportals#26]
I'm having the same issue, I get a successful log and the clients IP is added to "Authorized", but a page saying "You have not been authorized" displays, and when the client attempts to close, iOS asks to "Use Without Internet" Any suggestions? @kbeflo @H4xl0r 

Thank you, my Pineapple has an internet connection and works without the portal turned on.

_Originally posted by @ryanrohypnol in https://github.com/kbeflo/evilportals/issues/21#issuecomment-581147208_

**Comment by H4xl0r** [comment]
Still hanging there ? 
try my git of portals and review pls

**Comment by ryanrohypnol** [comment]
@H4xl0r I tried, same issue. It's probably something with my Pineapple.

**Comment by kbeflo** [comment]
Turns out I haven't included 

    <input type="hidden" name="target" value="<?=$destination?>">

On older portals

I included the fix on updated google-login, just checkout its branch.

Here's the current interaction using EvilPortalNano 3.2 and at firmware 2.7.

<img src="https://user-images.githubusercontent.com/13497504/82175509-a4aee400-9906-11ea-8b25-77b4ac1010b5.gif" height="500" />

**Comment by Crypteria** [comment]
I am new to the pretty new to the pineapple. I apologize in advance for my lack of knowledge in the area. I have downloaded the modules as instructed and am still getting the "you have not been authorized" messages. I saw another thread that someone just changed the default error message but I don't know how to do that, either. It looks as though the message still pops up in the image above if I'm not mistaken, so how can I fix it?
